css-positioning/fixed.css

```diff
@@ -0,0 +1,12 @@
+html {
+  font-size: 22px;
+}
+
+.page-title {
+  font-size: 3rem;
+  text-align: center;
+
+  /* your code here */
+  position: fixed;
+  left: calc(50% - (1rem * 10));
```

Comment:
Instead of doing this, you should set `width: 100%` so `text-align` can center it for you